README.md

```diff
@@ -25,6 +25,7 @@ ART is an open-source reinforcement training library for improving LLM performan
 | **2048**          | [🏋️ Train agent](https://colab.research.google.com/github/openpipe/art/blob/main/examples/2048/2048.ipynb)                   | Qwen 2.5 3B learns to play 2048           | <img src="https://github.com/openpipe/art/raw/main/assets/benchmarks/2048/accuracy-training-progress.svg" height="72"> [benchmarks](https://github.com/openpipe/art/blob/main/examples/2048/benchmark_2048.ipynb)                            |
 | **Temporal Clue** | [🏋️ Train agent](https://colab.research.google.com/github/openpipe/art/blob/main/examples/temporal_clue/temporal-clue.ipynb) | Qwen 2.5 7B learns to solve Temporal Clue | [Link coming soon]                                                                                                                                                                                                                           |
 | **Tic Tac Toe**   | [🏋️ Train agent](https://colab.research.google.com/github/openpipe/art/blob/main/examples/tic_tac_toe/tic-tac-toe.ipynb)     | Qwen 2.5 3B learns to play Tic Tac Toe    | <img src="https://github.com/openpipe/art/raw/main/assets/benchmarks/tic-tac-toe-local/accuracy-training-progress.svg" height="72"> [benchmarks](https://github.com/openpipe/art/blob/main/examples/tic_tac_toe/benchmark_tic_tac_toe.ipynb) |
+| **Codenames**     | [🏋️ Train agent](https://colab.research.google.com/github/openpipe/art/blob/add_codenames_nb/examples/codenames/Codenames_RL.ipynb)     | Qwen 2.5 3B learns to play Codenames    | <img src="https://github.com/openpipe/art/raw/add_codenames_nb/assets/benchmarks/codenames/win_rate_over_time.png" height="72"> [benchmarks](https://github.com/openpipe/art/blob/add_codenames_nb/examples/codenames/Codenames_RL.ipynb) |
```

Comment:
Could you save the benchmark image in the top-level `assets` folder for consistency?

Also, we're now using relative links to reference benchmark notebooks and images in this table (my fault for using absolute urls before). Could you merge in main and use the relative links here as well?